src/graph.ts

```diff
@@ -587,8 +666,13 @@ export class FactoryGraph {
      */
     createOutput(item: Craftable, outputRate: PerMinute, maintainedOutput: Quantity): OutputNode {
         const containers = this.getContainers(item)
-        const name = item.name + " C" + containers.size
-        const output = new OutputNode(name, item, outputRate, maintainedOutput, this)
+        const output = new OutputNode(
+            "C" + containers.size,
```

Comment:
```suggestion
            `C${containers.size}`,
```

Reply:
Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/e3612727cf42e8f92e923180d0c72033bd0a1513